### ISSUE TITLE
time series data is not recognized - empty chart is rendered

### ISSUE THREAD

**Issue by SrancsikAtBitrise** [metabase/metabase#10862]
**Describe the bug**
Time series data are not recognized when rendering charts. The year is set back to 1970.

**To Reproduce**
This is the data. Before upgrading to 0.33.1 chart was rendered properly.
![Screenshot 2019-09-11 at 11 49 18](https://user-images.githubusercontent.com/39155393/64687212-5cdbe100-d48a-11e9-9233-c710b6e85b45.png)

**Expected behavior**
Time series is recognized, data is ordered on X with proper

**Screenshots**
Nor `ordinal`, nor `time series` settings works.
![Screenshot 2019-09-11 at 11 43 48](https://user-images.githubusercontent.com/39155393/64687334-93196080-d48a-11e9-8f25-76122bd17f95.png)
![Screenshot 2019-09-11 at 11 43 32](https://user-images.githubusercontent.com/39155393/64687338-96ace780-d48a-11e9-9c1c-892d94b671c3.png)

**Information about your Metabase Installation:**
- Your browser and the version: Version 76.0.3809.132 (Official Build) (64-bit)
- Your operating system: macOS 10.14.6
- Your databases: Postgres
- Metabase version: 0.33.2
- Metabase hosting environment: Heroku
- Metabase internal database: Postgres

**Severity**
We can switch to tables but it is neither trivial nor informative in all the cases.

**Comment by Intorp** [comment]
We had the same issue, the problem seems to be the **Linear Inrerpolated** on the **Replace missing values with** Display option. We ended up reverting from 0.33.2 to 0.33.0 to fix this problem.

**Comment by paulrosenzweig** [comment]
I've been unable to reproduce this on v0.33.2. Can you provide a sample query that triggers this behavior? Either using the sample dataset or a simple SQL query like this would be great!
```sql
select '2019-09-10T01:23:45Z'::timestamptz as t, 1 as x1, 1 as x2
union all
select '2019-09-09T01:23:45Z'::timestamptz as t, 1 as x1, 1 as x2
union all
select '2019-09-07T01:23:45Z'::timestamptz as t, 1 as x1, 1 as x2

```

**Comment by helmecke** [comment]
If the first column is the date field its works but not otherwise.

```
select count(*) ,'2019-09-10T01:23:45Z'::timestamptz as t, 1 as x1, 1 as x2
union all
select count(*) ,'2019-09-09T01:23:45Z'::timestamptz as t, 1 as x1, 1 as x2
union all
select count(*) ,'2019-09-07T01:23:45Z'::timestamptz as t, 1 as x1, 1 as x2
```

**Comment by paulrosenzweig** [comment]
@helmecke I think you're seeing #10818, which will be fixed in the next release.

In the screenshot from @SrancsikAtBitrise's original comment, it looks like the date was in the first column so I think this is another issue.

**Comment by paulrosenzweig** [comment]
@SrancsikAtBitrise is there anymore info you can provide to help reproduce this? Is it related to the "Replace missing values with" for you as @Intorp mentioned? Thanks!

**Comment by SrancsikAtBitrise** [comment]
@paulrosenzweig I cannot confirm that nor the opposite.
Meanwhile, we downgraded to 0.32.4, that is working.

**Comment by paulrosenzweig** [comment]
Looking again at your initial screenshot, I noticed the refresh icon.

![image](https://user-images.githubusercontent.com/691495/65435682-e5457480-ddee-11e9-8ece-9fe1e3fb2105.png)

Was created_at not the first column in the result? If so, then I suspect @helmecke was right. If this is another form of #10818, then upgrading to v0.33.3 will fit this for you. Even if it's not, it's probably worth trying the new version to see if it fixes this.

**Comment by marcellocaron** [comment]
I was able to emulate this problem. It seems to be a problem with the alias.
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/44516854/66229853-28d69300-e6b9-11e9-92b4-3a872c490b31.gif)

**Comment by flamber** [comment]
@marcellocaron 
Which version of Metabase?
I think it's not the same problem. Notice that the series legend matches on the original issue, whereas it changes on your animation.
Visualization settings are stored based on column names, and when you change the column from all uppercase, then the visualization is being reset, which means you need to adjust which columns you're showing on the X-axis in Visualization Settings > Data.

**Comment by marcellocaron** [comment]
@flamber you are right. We are using 0.33.2.

**Comment by flamber** [comment]
@marcellocaron Upgrade to 0.33.3 - several issues with histogram charts were fixed.

**Comment by flamber** [comment]
I cannot reproduce this on 0.35.3, so I guess it was fixed in 0.33.3
